tokens/src/core/components/ActionRow.json

```diff
@@ -1,9 +1,10 @@
 {
+  "$type": "dimension",
```

Comment:
[clarification] Will _all_ tokens defined for `ActionRow` be of `"$type": "dimension"`? Should this be associated more directly to the dimension-specific tokens, even though these are the only tokens in this file right now?

For example, if/when a non-dimension token is added to this file, is there a risk this `$type` won't be semantically correct/accurate anymore?

Similar question like applies to other components' tokens, too.

Reply:
Resolved in https://github.com/openedx/paragon/pull/3207